src/icon/icon.module.ts

```diff
@@ -106,7 +105,6 @@ export class IconModule {
 			Copy20,
 			Data216,
 			Data220,
-			Delete16,
```

Comment:
V11 removes Delete icon & the Trash Can icon is its replacement, so should I import that instead?

Reply:
And replace it wherever is used (table story)